distrobox-enter

```diff
@@ -317,9 +317,9 @@ generate_command() {
 		# Skipping workdir we just enter $HOME of the container.
 		workdir="${container_home}"
 	fi
-	result_command="${result_command}
-		--workdir=\"${workdir}\"
-		--env \"CONTAINER_ID=${container_name}\"
+	result_command="${result_command} \\
+		--workdir=\"${workdir}\" \\
+		--env \"CONTAINER_ID=${container_name}\" \\
```

Comment:
Could we do this without doing this double backslash?
I'd like to keep code readability and make it a bit less tricky

Reply:
One way is to connect all the lines in a single line, which looks somewhat ugly.

```bash
  result_command="${result_command} --workdir=\"${workdir}\" --env \"CONTAINER_ID=${container_name}\" --env \"DISTROBOX_ENTER_PATH=${distrobox_enter_path}\""
```

Another way is to add an argument to `result_command` in each line separately.

```bash
  result_command="${result_command} --workdir=\"${workdir}\""
  result_command="${result_command} --env \"CONTAINER_ID=${container_name}\""
  result_command="${result_command} --env \"DISTROBOX_ENTER_PATH=${distrobox_enter_path}\""
```

Or another way is to use the line continuation (with backslashes outside the quotation), though the readability is bad:

```
  result_command="${result_command} "\
    "--workdir=\"${workdir}\" "\
    "--env \"CONTAINER_ID=${container_name}\" "\
    "--env \"DISTROBOX_ENTER_PATH=${distrobox_enter_path}\""
```

Reply:
I added a commit 88d6ac132179ab3cb9e06b3888905bdcf052387b that implements the second option.

Reply:
Perfect!